_config_dev.yml

```diff
@@ -0,0 +1,5 @@
+# For local development and testing, use the option
```

Comment:
I would remove this file as it does not contain any non-default setting.

Reply:
Thanks for the review! I agree that this file should be removed – especially as [a change in Jekyll v4](https://github.com/jekyll/jekyll/pull/8620) makes it useless.

This PR is included in #578, which @mattxwang is currently reviewing for possible inclusion in v0.4.0; if it is included, this PR will be closed, so I'm inclined to wait with updates until that has been decided.